docs/application/native/guides/index.md

```diff
@@ -22,18 +22,18 @@ The following table defines the Tizen mobile native API layout. The table descri
+| [Application Framework](../api/mobile/latest/group__CAPI__APPLICATION__FRAMEWORK.html) | [Application](../api/mobile/latest/group__CAPI__APPLICATION__MODULE.html) | Manages the application's main event loop, state change events, and basic system events, and gets information about the application. It is also used to launch other applications. | The Application Framework API contains submodule APIs for application development. The submodule APIs enable application life-cycle management, usage of functionality that is exported by other applications, and access to a user's application preferences.<br>Guides:<br>  - [Alarms and Scheduling](alarm/overview.md)<br>  - [Application Management](app-management/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md) |
+| | [Application Manager](../api/mobile/latest/group__CAPI__APPLICATION__MANAGER__MODULE.html) | Gets information, such as the application ID and path to the shared data directory, of the current application and other installed or running applications. | The Application Framework API contains submodule APIs for application development. The submodule APIs enable application life-cycle management, usage of functionality that is exported by other applications, and access to a user's application preferences.<br>Guides:<br>  - [Alarms and Scheduling](alarm/overview.md)<br>  - [Application Management](app-management/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md)  |
+| | [Attach Panel (since 2.4)](../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html) | Provides a panel in which users can take pictures, record voice, and select files to attach. | The Application Framework API contains submodule APIs for application development. The submodule APIs enable application life-cycle management, usage of functionality that is exported by other applications, and access to a user's application preferences.<br>Guides:<br>  - [Alarms and Scheduling](alarm/overview.md)<br>  - [Application Management](app-management/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md)  |
+| | [Badge](../api/mobile/latest/group__BADGE__MODULE.html) | [Creates and removes badges on the application's home screen icon. A badge is an image displayed on the application icon, which informs the user about notifications and events. This submodule can also be used to set and get the badge value and visibility.](app-management/app-icons.md) | The Application Framework API contains submodule APIs for application development. The submodule APIs enable application life-cycle management, usage of functionality that is exported by other applications, and access to a user's application preferences.<br>Guides:<br>  - [Alarms and Scheduling](alarm/overview.md)<br>  - [Application Management](app-management/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md)  |
```

Comment:
Just for check. Every sentences in the second cell are linked. Is it ok?
![image](https://user-images.githubusercontent.com/9958374/48822833-ca0a7280-eda1-11e8-9658-cc0d70a30d2f.png)

Reply:
I removed this link and the link for libXML because they were the only descriptions having links. 

BTW, I think this table is hard to understand. I will change this table to list or other tags later.